spec/rest-middleware.spec.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+describe("restMiddleware", () => {
+    var configMock, helmetMock, logger, restMiddleware, serverMock, restifyLinks;
+
+    beforeEach(() => {
+        var LoggerMock, RestMiddleware;
+
+        RestMiddleware = require("../lib/rest-middleware");
+        LoggerMock = require("./mock/logger-mock");
+        helmetMock = jasmine.createSpyObj("helmetMock", [
+            "frameguard",
+            "hidePoweredBy",
+            "hsts",
+            "ieNoOpen",
+            "noCache",
+            "noSniff",
+            "xssFilter"
+        ]);
+        serverMock = jasmine.createSpyObj("serverMock", [
+            "use"
+        ]);
+        configMock = {
```

Comment:
Additionally, it looks like you can just use `{}` or a hardcoded config instead of this config object in most places.  I would prefer that as it more distinctly calls out the bits that you need for the function to exhibit the desired behavior.